hack/sandboxed.Dockerfile

```diff
@@ -0,0 +1,31 @@
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM golang:alpine as builder
+
+WORKDIR /go/src/sigs.k8s.io/krew
+
+# build binary
+COPY . .
+RUN go build -ldflags "-s -w" ./cmd/krew
```

Comment:
can't we use the existing hack/ script to build here?

Reply:
The original build script uses `gox` for the build and does a cross-compile for all kinds of platforms. I would rather not add `gox` as another dependency in the builder, because the target is a single platform. Besides, I'm not sure if `gox` links correctly againgst `libc.musl` here.

There is of course one drawback: the gitCommit and version will not be set up correctly. I think that's affordable though.

Reply:
#180 proposed we can move away from gox. The only thing we get out of gox is parallelism but it’s not absolutely necessary.

I think it’s ok to not to have the gitCommit inside the container, although I guess you can take it out of .dockerignore and that’d work, too?

Reply:
We can do it later.